src/ng-generate/table/index.ts

```diff
@@ -589,9 +594,13 @@ function updateConfigFiles(options: any): Rule {
         addStylePreprocessorOptions(angularBuildOptions);
 
         const defaultMaterialtheme = 'node_modules/@angular/material/prebuilt-themes/indigo-pink.css';
-
-        if (!angularBuildOptions['styles'].includes(defaultMaterialtheme)) {
-            angularBuildOptions['styles'].push(defaultMaterialtheme);
+        if (options.getOptionalMaterialTheme) {
+            if (!angularBuildOptions['styles'].includes(defaultMaterialtheme)) {
+                angularBuildOptions['styles'].push(defaultMaterialtheme);
+            }
+        } else if (!options.getOptionalMaterialTheme && angularBuildOptions['styles'].includes(defaultMaterialtheme)) {
+            const index = angularBuildOptions['styles'].indexOf(defaultMaterialtheme);
+            angularBuildOptions['styles'].splice(index, 1);
```

Comment:
Please, keep all blocks consistent:
```
if (options.getOptionalMaterialTheme && !angularBuildOptions['styles'].includes(defaultMaterialtheme)) {
  angularBuildOptions['styles'].push(defaultMaterialtheme);
} else if (!options.getOptionalMaterialTheme && angularBuildOptions['styles'].includes(defaultMaterialtheme)) {
  const index = angularBuildOptions['styles'].indexOf(defaultMaterialtheme);
  angularBuildOptions['styles'].splice(index, 1);
}
```
Is it possible to use `else` here instead of specifying another condition?
```
if (options.getOptionalMaterialTheme && !angularBuildOptions['styles'].includes(defaultMaterialtheme)) {
  angularBuildOptions['styles'].push(defaultMaterialtheme);
} else {
  const index = angularBuildOptions['styles'].indexOf(defaultMaterialtheme);
  angularBuildOptions['styles'].splice(index, 1);
}
```

Reply:
Good question, can you test it @larisaVasile?

Reply:
Another one question is why we also need to delete here conditionally, isn't it only about adding a new style to the list? Simply like this:
```
if (options.getOptionalMaterialTheme && !angularBuildOptions['styles'].includes(defaultMaterialtheme)) {
  angularBuildOptions['styles'].push(defaultMaterialtheme);
}
```
Since there was no logic for removal before, why did it become necessary?

Reply:
hi, i had a discussion with Anton and i removed the else if.  I added that else if to remove the theme is case it was one added by the user and getOptionalMaterialTheme is false. But it is better that if getOptionalMaterialTheme is false than we do nothing.